src/app/content/components/Toolbar/styled.tsx

```diff
@@ -376,3 +361,20 @@ export const SidebarControl = styled(OpenSidebarControl)`
 export const NudgeElementTarget = styled.div`
   display: contents;
 `;
+
+// tslint:disable-next-line:variable-name
```

Comment:
I've moved it down because `SidebarControl` has to be defined before being used.